build.gradle.kts

```diff
@@ -265,7 +293,7 @@ val javadoc by tasks.getting(Javadoc::class) {
 
         author()
         tags("incubating:a:Incubating:")
-        links("https://docs.oracle.com/javase/8/docs/api/", "https://takahikokawasaki.github.io/nv-websocket-client/")
+        links("https://docs.oracle.com/en/java/javase/$currentJavaVersion/docs/api/", "https://takahikokawasaki.github.io/nv-websocket-client/")
```

Comment:
Might also want to add the `--no-fonts` option, this is new stuff that takes about 4 MB

Reply:
Given that this is just docs, I don't really see a problem with a bigger jar size for the fonts. Since we don't have our own custom fonts and want to just use the default.